packages/status-js/package.json

```diff
@@ -54,5 +50,8 @@
   ],
   "publishConfig": {
     "access": "public"
+  },
+  "browser": {
+    "./src/utils/encode-url-data": false
```

Comment:
Excludes `zlib` dependent code from client bundle.

Reply:
Other approach like `resolve.alias.zlib: `./my-custom-zlib-throwing-errors-if-called.ts'` in the project that depends on `@status-im/js` also worked.

Reply:
The Next.js server-code (e.g. `getServerSideProps`) will correctly run in the Node environment, so it should still work correctly.